packages/hash/subgraph/src/stdlib/edge/link.ts

```diff
@@ -1,12 +1,27 @@
 import { Subgraph } from "../../types/subgraph";
-import { EntityId } from "../../types/identifier";
+import { entityEditionIdToString, EntityId } from "../../types/identifier";
 import { Entity } from "../../types/element";
 import { getEntityAtTimestamp } from "../element/entity";
 import {
   isOutwardLinkEdge,
   isHasRightEntityEdge,
+  isHasLeftEntityEdge,
 } from "../../types/edge/outward-edge-alias";
 import { mustBeDefined } from "../../shared/invariant";
+import { OutwardEdge } from "../../types/edge";
+
+const getUniqueEntitiesFilter = () => {
```

Comment:
<img width="667" alt="image" src="https://user-images.githubusercontent.com/25749103/204002673-a67305c6-2b03-44ce-a8e3-cdc3e905c9c8.png">

I have fixed the duplication we're seeing in main on edges.

There is now only one Org in the dropdown
<img width="261" alt="image" src="https://user-images.githubusercontent.com/25749103/204002940-c302878f-0ad9-4b23-a4bc-8d3e7712f425.png">